README.md

```diff
@@ -1 +1,70 @@
-# HELLO-MICROVERSE1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **BADRU NDAWULA**
+
+- GitHub: [@PrinceBadru]
```

Comment:
- [ ] Great Job Creating Readme.md file, but the template has been updated and you need to use the [new one](https://github.com/microverseinc/readme-template) , kindly update it and make sure to fill in the necessary fields 🙏

Reply:
Just as pointed out by the previous reviewer, the readme template has been updated you can find it [here](https://github.com/microverseinc/readme-template). Kindly update your readme file using this template as it is required henceforth to use the new template in order to receive reviews.

Here's how it looks:
![image](https://user-images.githubusercontent.com/20554583/204871202-8706df00-b768-42e1-b2c4-a37494b41ad5.png)